android/tangram/jni/platform_android.cpp

```diff
@@ -112,32 +115,64 @@ bool isContinuousRendering() {
 
 }
 
-std::string stringFromResource(const char* _path) {
+std::string setResourceRoot(const char* _path) {
+
+    s_resourceRoot = std::string(dirname(_path));
 
-    std::string out;
+    s_useInternalResources = (*_path != '/');
+
+    if (s_resourceRoot.front() == '.') {
```

Comment:
According to the documentation for `string::front`: "This function shall not be called on empty strings.". Can we do a check for that first?